tests/it/language/java_11.rs

```diff
@@ -0,0 +1,105 @@
+use pretty_assertions::assert_eq;
+use snippets::{language::java_11, Extractor, Kind, Location, Metadata, Method, Snippet};
+
+use crate::include_str_lf;
+
+// #[test]
```

Comment:
[Prefer] Let's kill dead comments or add one-two line explaining why we are keeping this test commented!